### PR TITLE
Avoid white flick when launching the app

### PR DESCRIPTION
**Pull request by Flaburgan**
Display a loading above everything else and then remove it once the chatList or the registration page is ready. Is there other entry points?

### CODE REVIEW

**Comment by nanu-c** [comment]
Does this work with dark mode and light mode?

**Comment by Flaburgan** [comment]
Yes it does. The way it works, there is an hardcoded "loading" screen in front of everything right in index.html so loaded immediately. Then when the app is ready (`mounted` event in either chatList or Register, the two entry points I am aware of), that screen is removed and the loaded app behind it appears.

**Comment by nanu-c** [comment]
You have to add it also to the `axolotl-web/src/pages/Password.vue` because in case of an encrypted storage this is the entry point

File: axolotl-web/src/pages/Register.vue [hunk 1]
```diff
@@ -61,6 +61,10 @@ export default {
     var userLang = navigator.language || navigator.userLanguage;
     this.$language.current = userLang;
     document.getElementById("VuePhoneNumberInput_phone_number").focus();
+    let loader = document.getElementById('initial-loader');
```
Comment (Flaburgan) [inline axolotl-web/src/pages/Register.vue:64]: @nanu-c Is there a way to make a function global so I don't duplicate it everywhere?

**Comment by Flaburgan** [comment]
> You have to add it also to the `axolotl-web/src/pages/Password.vue` because in case of an encrypted storage this is the entry point

Done.

File: axolotl-web/src/pages/Register.vue (hunk 1 above)
Comment (nanu-c) [reply axolotl-web/src/pages/Register.vue:64]: make a component and include it in the header `axolotl-web/src/components/Header.vue`  and then it should work.

**Comment by Flaburgan** [comment]
https://github.com/nanu-c/axolotl/pull/265 is merged so I am closing this.

**Pull request closed without merging**